client/src/components/common/card/ReviewCard.jsx

```
@@ -20,27 +21,29 @@ export default function ReviewCard({ reviewDetail, setUserInfo, size = 'small' }
+          review={review}
+          user={user}
+          bookId={book.id}
+          toggleCard={toggleCard}
+          setUserInfo={setUserInfo}
+          isDelete={isDelete}
+          setIsDelete={setIsDelete}
+          isOpen={isOpen}
+          setIsOpen={setIsOpen}
+          toggleCardHandler={toggleCardHandler}
+          clickProfileHandler={clickProfileHandler}
+        ></CardFront>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)